test/whitelist.spec.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-unused-expressions */
+
+
+const faker = require('faker')
+const { getConfig } = require('../config')
+const XTokenTZ = require('./helpers/token')
+
+describe("adding accounts to WhiteListing category" ,()=>{
+
+  let storage
+  let config
+  let accounts
+
+  before(async () => {
+    config = await getConfig()
+    accounts = config.accounts
+    const { alice } = accounts
+
+    security = await XTokenTZ.originate(alice) //adding alice as the owner or admin
+    storage = await security.storage()
+    let bob_id; 
```

Comment:
Unused variables `bob_id` and `eve`

Reply:
bob_id and eve are being used in       bob_id- line 28,line46  eve- line 40, line 42, line54

Reply:
correct, my bad